components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { action } from "@storybook/addon-actions";
+import Card from "./Card";
+
+export default {
+  title: "Card",
+  component: Card
+};
+
+export const card = () => (
+  <Card
+    onClick={action("clicked")}
+    style={{ width: "500px", height: "150px" }}
+  />
+);
```

Comment:
We need some example that what looks if you use `<Card>` without styles (=as default).

Reply:
<img width="1227" alt="Screen Shot 2020-03-16 at 11 01 23 AM" src="https://user-images.githubusercontent.com/35892492/76787029-7e898e00-6775-11ea-8efa-a698955277d0.png">

should I put some default style?